src/AccessibilityInsights.SharedUx/Settings/ConfigurationModel.cs

```diff
@@ -126,8 +126,8 @@ public string AppVersion
 
         public Guid SelectedIssueReporter
         {
-            get => GetDataValue<Guid>(keyIssueReporting);
-            set => SetDataValue<Guid>(keyIssueReporting, value);
+            get => Guid.Parse(GetDataValue<string>(keyIssueReporting));
+            set => SetDataValue<string>(keyIssueReporting, value.ToString());
```

Comment:
>ToString() [](start = 65, length = 10)

Guid.ToString supports an optional formatting parameter (see [https://docs.microsoft.com/en-us/dotnet/api/system.guid.tostring?view=netframework-4.7.2](https://docs.microsoft.com/en-us/dotnet/api/system.guid.tostring?view=netframework-4.7.2)). Are we OK with the default format?

Reply:
Guid.Parse should work regardless of format, so I think we're fine with the default:
>This method can convert strings in any of the five formats produced by the ToString
- https://docs.microsoft.com/en-us/dotnet/api/system.guid.parse?view=netframework-4.7.2